src/main/java/com/github/stickerifier/stickerify/bot/Stickerify.java

```diff
@@ -97,32 +96,35 @@ private void answerFile(TelegramRequest request) {
 	}
 
 	private File retrieveFile(String fileId) throws TelegramApiException {
-		GetFile getFile = new GetFile(fileId);
+		var file = execute(new GetFile(fileId)).file();
```

Comment:
Shouldn't we check for `.isOk()` here as well?

Reply:
Yep, you are right! I will amend the code accordingly